GitUI/CommandsDialogs/FormPush.cs

```diff
@@ -150,10 +153,40 @@ public DialogResult PushAndShowDialogWhenFailed()
             return PushAndShowDialogWhenFailed(null);
         }
 
+
+        private bool CheckRemotePresent()
+        {
+            if (_gitRemoteController.Remotes.Count < 1)
+            {
+                if (DialogResult.Yes == MessageBox.Show(this, _configureRemote.Text, _errorCaption.Text, MessageBoxButtons.YesNo, MessageBoxIcon.Error))
+                {
+                    // launch remote management dialog
+                    ManageRemotes(null);
+                    return _gitRemoteController.Remotes.Count > 0;
+                }
+                return false;
+            }
+            if (_selectedRemote == null || _NO_TRANSLATE_Remotes.SelectedIndex < 0)
+            {
+                MessageBox.Show(this, _selectRemote.Text, _errorCaption.Text, MessageBoxButtons.OK, MessageBoxIcon.Error);
+                return false;
+            }
+            return true;
+        }
+
+        private void ManageRemotes(string selectedRemoteName)
+        {
+            // launch remote management dialog
+            UICommands.StartRemotesDialog(this, selectedRemoteName);
+
+            // coming back from the dialog, update the remotes list
+            _gitRemoteController.LoadRemotes(false);
```

Comment:
Extract to method and remove comment

Reply:
what exactly needs extracting?

Reply:
using comment is not a very good practice to describe what the code do (should be kept to explain why).
If needed, that's better to put the code in a method describing what is done (and let the compiler inline the code).
So, I think, that's better to put the 2 lines in a method (why not called `UpdateRemoteList()`?).

Reply:
I totally agree, comments should describe 'why' (if necessary), and not 'what'.

The comments were existing artifacts (which I have probably introduced originally... /facepalm )